services/provider/server/storageclaim.go

```diff
@@ -64,6 +64,10 @@ func (s *storageClassClaimManager) Create(ctx context.Context, consumer *ocsv1al
 			Name:            generatedClaimName,
 			Namespace:       s.namespace,
 			OwnerReferences: []metav1.OwnerReference{ownerRef},
+			Annotations: map[string]string{
+				consumerUUIDAnnotation:          consumerUUID,
+				storageClassClaimNameAnnotation: storageClassClaimName,
```

Comment:
Why do we need this?

Reply:
This is for tracking purposes, adding the claimName as metadata to the claim CR.